gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/platform/LicenseResource.java

```diff
@@ -51,7 +51,7 @@ public void get(@Suspended final AsyncResponse response) {
                 .tier(platformLicense.getTier())
                 .packs(platformLicense.getPacks())
                 .features(platformLicense.getFeatures())
-                .expirationDate(isLicenseExpirationNotifierEnabled ? platformLicense.getExpirationDate() : null)
+                .expiresAt(isLicenseExpirationNotifierEnabled ? platformLicense.getExpirationDate() : null)
```

Comment:
Non-blocking: It would be best to refactor `isLicenseExpirationNotifierEnabled` into another endpoint. That way we can return the true expiration date of the license with this endpoint, and can recuperate the info if the license notification is enabled from another endpoint.